Assets/HoloToolkit/Sharing/Tests/Scripts/RemoteHeadManager.cs

```diff
@@ -137,7 +138,7 @@ private void UpdateHeadTransform(NetworkInMessage msg)
             Quaternion headRot = CustomMessages.Instance.ReadQuaternion(msg);
 
             RemoteHeadInfo headInfo = GetRemoteHeadInfo(userID);
-            headInfo.HeadObject.transform.localPosition = headPos;
+            headInfo.HeadObject.transform.localPosition = headPos += new Vector3(0,-1.0f,0);
```

Comment:
Yes, 1 meter below. I should actually lower it by about 2m - since the 0,0,0 position of the avatar is the feet - and we want the feet to be at ground-level instead of head-level.